tests/integration_tests/main.go

```diff
@@ -32,8 +28,7 @@ func handleRequest(ctx context.Context, ev events.APIGatewayProxyRequest) (event
 	client.Do(req)
 
 	// Metric
-	ddlambda.Distribution("hello-go.dog", float64(invokeCount))
-	invokeCount++
```

Comment:
This breaks the snapshot when running the tests consecutively which reuses the same lambda function runtime, and the count would increase for each run.